alpine-aws/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# dkubb/alpine-aws
+
+FROM alpine:3.6
+MAINTAINER Dan Kubb <dkubb@fastmail.com>
+
+# Upgrade system dependencies
+RUN apk upgrade --update-cache --available
+
+# Install system dependencies
+RUN apk add --update-cache py2-pip=9.0.1-r1
```

Comment:
@mbj ok, I've merged this, but if you want you can push a new PR with the removal.